apps/win/vscode.talon

```diff
@@ -6,6 +6,31 @@ app: Visual Studio Code
 app: Code.exe
 -
 
+# General
+action(user.ide_command_palette):
+  key(ctrl-shift-p)
+
+# Splits
+action(user.ide_split_right):
+  key(ctrl-shift-p)
```

Comment:
I'd suggest calling the user.ide_command_palette() here, so if the shortcut's just in one place.

Reply:
Agreed